src/main/kotlin/com/chutneytesting/kotlin/launcher/Launcher.kt

```diff
@@ -0,0 +1,211 @@
+package com.chutneytesting.kotlin.launcher
+
+import com.chutneytesting.ExecutionConfiguration
+import com.chutneytesting.engine.api.execution.*
+import com.chutneytesting.engine.api.execution.StatusDto.SUCCESS
+import com.chutneytesting.environment.api.EmbeddedEnvironmentApi
+import com.chutneytesting.environment.domain.EnvironmentService
+import com.chutneytesting.environment.infra.JsonFilesEnvironmentRepository
+import com.chutneytesting.kotlin.dsl.*
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import org.junit.jupiter.api.Assertions.assertAll
```

Comment:
do we really want to be dependant to JUNIT5 ?
Assertj is less impacting as a dependency than junit.